src/service/UserService.ts

```diff
@@ -21,18 +24,24 @@ export default class UserService {
         }
     }
 
-    public async login(loginDTO : LoginDTO) : Promise<_User> {
+    public async login(loginRequestDTO : LoginRequestDTO) : Promise<LoginResponseDTO> {
```

Comment:
This now expects a LoginRequestDTO and will return a LoginResponseDTO. I think we should follow this pattern?